build/ci-build.yml

```
@@ -0,0 +1,122 @@
+name: Build Arcus.Security CI
+
+trigger:
+  - master
+pr:
+  - master
+
+variables:
+  - group: 'Arcus Security - Integration Testing'
+  - group: 'Arcus - GitHub Package Registry'
+  - group: 'Build Configuration'
+
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.105'
+  - name: 'Project'
+    value: 'Arcus.Security'
+
+# Building the project:
+# - Download correct .NET Core SDK
+# - .NET Core build
+# - Publish build artifacts
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration $(Build.Configuration)'
+          - task: CopyFiles@2
+            displayName: 'Copy build artifacts'
+            inputs:
+              contents: '**/?(bin|obj)/**'
+              targetFolder: '$(Pipeline.Workspace)/build'
+          - task: PublishPipelineArtifact@0
+            displayName: 'Publish build artifacts'
+            inputs:
+              targetPath: '$(Pipeline.Workspace)/build'
+              artifactName: Build
+
+  # Testing unit tests of the project:
+  # - Download build artifacts
+  # - Download correct .NET Core SDK
+  # - .NET Core test
+  - stage: UnitTests
+    dependsOn: Build
```

Let's add a display name maybe?

```suggestion
    displayName: Unit Tests
    dependsOn: Build
```